ipa-core/src/protocol/ipa_prf/boolean_ops/comparison_and_subtraction_sequential.rs

```diff
@@ -32,11 +37,11 @@ where
     C: Context,
     YS: SharedValue + CustomArray<Element = XS::Element>,
     XS: SharedValue + CustomArray + Field,
-    XS::Element: Field + std::ops::Not<Output = XS::Element>,
+    XS::Element: Field,
+    AdditiveShare<XS::Element>: std::ops::Not<Output = AdditiveShare<XS::Element>>,
```

Comment:
should we consider implementing `Not` for `AdditiveShare<CustomArray>`?

Reply:
yea i was suggesting to implement Not for shares of boolean arrays only. If we don't need that, it is even better